### ISSUE TITLE
MultipleStatementAlignment breaks on comment in the end of an array

### ISSUE THREAD

**Issue by ComiR** [squizlabs/PHP_CodeSniffer#3175]
**Generic.Formatting.MultipleStatementAlignment**
If an array contains a comment as the last element, the alignment of the assignments breaks (wrong warnings and fixes).
#1848 already fixed this for comments not on the last line.

**Code sample**
```php
$foo = array(
    // Some comment before.
    'a' => 'b',
);
$barbar = 'bar';

$foo = array(
    'a' => 'b',
    // Some comment in between.
    'b' => 'c',
);
$barbar = 'bar';

$foo = array(
    'a' => 'b',
    // Some comment after.
);
$barbar = 'bar';
```

**Expected output**
```php
$foo    = array(
    // Some comment before.
    'a' => 'b',
);
$barbar = 'bar';

$foo    = array(
    'a' => 'b',
    // Some comment in between.
    'b' => 'c',
);
$barbar = 'bar';

$foo    = array(
    'a' => 'b',
    // Some comment after.
);
$barbar = 'bar';
```

**Actual output**
```php
$foo    = array(
    // Some comment before.
    'a' => 'b',
);
$barbar = 'bar';

$foo    = array(
    'a' => 'b',
    // Some comment in between.
    'b' => 'c',
);
$barbar = 'bar';

$foo = array(
    'a' => 'b',
    // Some comment after.
);
$barbar = 'bar';
```
